lib/cc/services/github_pull_requests.rb

```diff
@@ -40,30 +40,34 @@ def receive_test
 
   def receive_pull_request
```

Comment:
@noahd1 - I think the new `#receive_pull_request` is easier to understand. wdyt?